include/net/netfilter/nf_tables.h

```diff
@@ -1561,4 +1562,17 @@ __printf(2, 3) int nft_request_module(struct net *net, const char *fmt, ...);
 #else
 static inline int nft_request_module(struct net *net, const char *fmt, ...) { return -ENOENT; }
 #endif
+
+struct nftables_pernet {
+	u64                     tstamp;
+	unsigned int            gc_seq;
+};
```

Comment:
This is teachincally not in `4.18.0-534`
https://github.com/ctrliq/kernel-src-tree/blame/centos_kernel-4.18.0-534.el8/include/net/netfilter/nf_tables.h#L1535

The do however bring in the `tstamp` variable later as in `4.18.0-553.7.1` they do introduce
https://github.com/torvalds/linux/commit/7395dfacfff65e9938ac0889dafa1ab01e987d15#diff-71d63f69cc4d2a03182dc5a2faafca4785546e8e6238d5b1b4f543f80a3901ceR1803
```
* Thu Jun 06 2024 Denys Vlasenko <dvlasenk@redhat.com> [4.18.0-553.7.1.el8_10]
- netfilter: nf_tables: use timestamp to check for set element timeout (Phil Sutter) [RHEL-38023] {CVE-2024-27397}

* Wed May 29 2024 Denys Vlasenko <dvlasenk@redhat.com> [4.18.0-553.6.1.el8_10]
```

Reply:
Dropping that.

Reply:
Dropped the inline function that used that tstamp field. That inline function is never used.